src/DynamoCore/DynamoCore.csproj

```diff
@@ -1,184 +1,184 @@
-    <UseVSHostingProcess>false</UseVSHostingProcess>
-    <DocumentationFile>$(OutputPath)\DynamoCore.XML</DocumentationFile>
-    <LibGOsToken>windows</LibGOsToken>
-    <LibGOsToken Condition="$(Platform.Contains('Linux'))">linux</LibGOsToken>
+	<ProjectGuid>{7858FA8C-475F-4B8E-B468-1F8200778CF8}</ProjectGuid>
+	<OutputType>Library</OutputType>
+	<AppDesignerFolder>Properties</AppDesignerFolder>
+	<RootNamespace>Dynamo</RootNamespace>
+	<AssemblyName>DynamoCore</AssemblyName>
+	<UseVSHostingProcess>false</UseVSHostingProcess>
+	<DocumentationFile>$(OutputPath)\DynamoCore.XML</DocumentationFile>
+	<LibGOsToken>windows</LibGOsToken>
+	<LibGOsToken Condition="$(Platform.Contains('Linux'))">linux</LibGOsToken>
   </PropertyGroup>
   <PropertyGroup>
-    <NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
+	<NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
   </PropertyGroup>
-    <ItemGroup>
-    <Compile Remove="Graph\ConnectorPinModel.cs" />
+	<ItemGroup>
+	<Compile Remove="Graph\ConnectorPinModel.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Win32.Registry" Version="4.7.0" />
-    <!--The System.Drawing package is only included in DynamoCore (and not in all projects that have image resources) for simplicity-->
-    <PackageReference Include="System.Drawing.Common" Version="6.0.0" />
+	<PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+	<PackageReference Include="Microsoft.Win32.Registry" Version="4.7.0" />
+	<!--The System.Drawing package is only included in DynamoCore (and not in all projects that have image resources) for simplicity-->
+	<PackageReference Include="System.Drawing.Common" Version="6.0.0" />
   </ItemGroup>
   <ItemGroup Label="Common dependencies">
-    <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
-    <PackageReference Include="Greg" Version="3.0.0.3175" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.3175" GeneratePathProperty="true" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.3241" GeneratePathProperty="true" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" CopyXML="true" />
-	<PackageReference Include="RestSharp" Version="108.0.1" CopyXML="true" />
+	<PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
+	<PackageReference Include="Greg" Version="3.0.0.3175" />
+	<PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="3.0.0.3175" GeneratePathProperty="true" />
+	<PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.3241" GeneratePathProperty="true" />
+	<PackageReference Include="Newtonsoft.Json" Version="13.0.1" CopyXML="true" />
+	<PackageReference Include="RestSharp" Version="109.0.1" CopyXML="true" />
```

Comment:
@mjkkirschner wasn’t there an issue with revit and this version of restsharp?

Reply:
using the same 108.0.1 version now